src/rpc/common/Validators.cpp

```diff
@@ -199,4 +199,73 @@ CustomValidator IssuerValidator =
         return MaybeError{};
     }};
 
+CustomValidator SubscibeStreamValidator =
```

Comment:
Put here for now. I will work on unsubscribe, I might move it to common place of subscribe and unsubscribe , like a base class